model-optimizer/extensions/front/tf/reduce_ext.py

```diff
@@ -27,6 +27,16 @@ def extract(cls, node: Node):
         return cls.enabled
```

Comment:
also, could you please add extractor for `SelectV2`? I think it should be easy

Reply:
and `Max`

Reply:
The operation semantic of `SelectV2` and `Select` is different. The shape inference in the TF are different so this may be not easy. Anyway this is a different story and requires additional tests so I don't want to make this change in this PR.

Reply:
agree